gensrc/thrift/FrontendService.thrift

```diff
@@ -294,6 +294,7 @@ struct TGetDbsParams {
   2: optional string user   // deprecated
   3: optional string user_ip    // deprecated
   4: optional Types.TUserIdentity current_user_ident // to replace the user and user ip
+  5: optional string filter_type
```

Comment:
Could you give some comment for this field.
Why not use enum for this type?

Reply:
I will remove this field and try get all task/taskrun once.